ucp/_libs/transfer_common.pyx

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2019-2020, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2020       UT-Battelle, LLC. All rights reserved.
+# See file LICENSE for terms.
+
+# cython: language_level=3
+
+from libc.stdint cimport uintptr_t
+
+from .ucx_api_dep cimport *
+
+from ..exceptions import UCXCanceled, UCXError, log_errors
+
+
+cdef void _send_callback(void *request, ucs_status_t status):
```

Comment:
I would be good a comment saying it is used by `stream_send_nb` and `tag_send_nb`

Reply:
Done in https://github.com/rapidsai/ucx-py/pull/716/commits/d45563064a5bd555c080a7e7a54ad8376e1e49df . Also `UCXEndpoint.flush` and `UCXWorker.flush` depend on it as well.